app/main.py

```diff
@@ -1,7 +1,34 @@
+import os
+
+import dotenv
+import requests
+
+
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    key = os.getenv("API_KEY")
```

Comment:
Make sure that the `.env` file contains the `API_KEY` variable, as it is necessary for the API request.